docker/web/Dockerfile

```diff
@@ -15,10 +17,6 @@ COPY virtualhost-ssl.conf /etc/apache2/sites-available/000-default-ssl.conf
 RUN a2enmod rewrite ssl
 
 # php
-RUN apt-get update && \
-    apt-get install -y \
-                apt-transport-https lsb-release ca-certificates \
-                wget unzip git
 RUN wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
 RUN echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list
```

Comment:
I personally prefer just to type _stretch_ instead of installing using _lsb_release_. Anyway, this is OK.